common-java-libs/org.wso2.apk.apimgt.rest.api.common/build.gradle

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+description = 'WSO2 APK API Common Component'
```

Comment:
We need to remove the Rest API Common Module and move that to Rest API Utils. Part of the code applicable to admin services were already moved to utils. Shall we move the newly added code also?

Reply:
Removed.